specificatie/filter.yaml

```diff
@@ -0,0 +1,30 @@
+openapi: 3.0.3
+info:
+  title: Filter definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  parameters:
+    Fields:
+      name: fields
+      in: query
+      required: true
+      schema:
+        $ref: '#/components/schemas/Fields'
+
+  schemas:
+    Fields:
+      description: |
+          Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een door komma's gescheiden lijst van property namen op te geven.
+          Bij opgave van niet-bestaande properties wordt een 400 Bad Request teruggegeven. Wanneer de fields parameter niet is opgegeven, worden alle properties met een waarde teruggegeven.
+          Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/fields.feature)
+      type: string
+      pattern: ^[a-zA-Z0-9\.,_]+$
+      maxLength: 924
+    StraatFilter:
+      description: |
+        Een naam die door de gemeente aan een openbare ruimte is gegeven. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).
+      type: string
+      pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{7,80}(\?+|\*)?$
```

Comment:
deze moet denk ik nog worden aangepast, want wanneer je geen wildcard gebruikt, moet het wel mogelijk zijn minder dan 7 tekens in te voeren. Bijvoorbeeld zoeken op korte straatnamen "Laan" (2512 GM) of "Spui" (2511 BT) zou anders niet mogelijk zijn

Reply:
zoiets: ^([a-zA-Z0-9À-ÿ '\-\']{1,80}|((\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{7,79})|([a-zA-Z0-9À-ÿ '\-\']{7,79}(\?+|\*)?))$